package.json

```diff
@@ -61,10 +64,12 @@
     "mocha-testdata": "^1.2.0",
     "nyc": "^15.0.1",
     "protractor": "^5.4.3",
+    "puppeteer": "^2.1.1",
     "rimraf": "^3.0.2",
     "semantic-release": "^17.0.4",
     "semantic-release-cli": "^5.2.3",
     "travis-deploy-once": "^5.0.11",
+    "ts-mocha": "^7.0.0",
```

Comment:
Let's stick to regular `mocha` for now and migrate to `ts-mocha` in a separate PR

Reply:
removed ts-mocha